src/Api/Orbit.cpp

```diff
@@ -47,7 +47,7 @@ void orbit_api_start_v1(const char* name, orbit_api_color color, uint64_t group_
 }
 
 [[deprecated]] void orbit_api_start(const char* name, orbit_api_color color) {
-  uint64_t return_address = ORBIT_GET_CALLER_PC();
+  auto return_address = ORBIT_GET_CALLER_PC();
```

Comment:
This looks worse to me, what's clang-tidy's reasoning?

Reply:
clang-tidy does expand the macro, but in this case I changed it back.

Reply:
Unrelated curious question. Would it be possible to replace the macro by a force-inlined function?

I guess we don't wanna mess with Orbit.h though.

Reply:
As discussed offline. Not going to tackle this now/here. We need to be a bit careful, as Orbit.h is supposed to be also compiled in C code (while we are not standard compliant here anyways...).